packages/js/product-editor/src/components/variations-table/shipping-menu-item/shipping-menu-item.tsx

```diff
@@ -239,6 +241,10 @@ export function ShippingMenuItem( {
 					>
 						{ __( 'Set weight', 'woocommerce' ) }
 					</MenuItem>
+					<VariationQuickUpdateMenuItem.Slot
+						group={ 'shipping' }
+						supportsMultipleSelection={ supportsMultipleSelection }
+					/>
```

Comment:
We need to pass through the `selection` and `onChange` so that fills can access that in their `onClick` handler and actually do something!

We should also pass  the`onClose` callbacks, as that will be needed too for extensions to close the menu when they have done their action!